docker/examples/stats/stats.go

```diff
@@ -1,38 +1,39 @@
 package main
 
 import (
-  "os"
-  "fmt"
-  "github.com/sendgrid/sendgrid-go"
-  "log"
+	"fmt"
+	"github.com/sendgrid/sendgrid-go"
+	"log"
+	"os"
 )
 
 ///////////////////////////////////////////////////
 // Retrieve global email statistics
 // GET /stats
 
-func Retrieveglobalemailstatistics() {
-  apiKey := os.Getenv("SENDGRID_API_KEY")
-  host := "http://localhost:4010"
-  request := sendgrid.GetRequest(apiKey, "/v3/stats", host)
-  request.Method = "GET"
-  queryParams := make(map[string]string)
-  queryParams["aggregated_by"] = "day"
-  queryParams["limit"] = "1"
-  queryParams["start_date"] = "2016-01-01"
-  queryParams["end_date"] = "2016-04-01"
-  queryParams["offset"] = "1"
-  request.QueryParams = queryParams
-  response, err := sendgrid.API(request)
-  if err != nil {
-    log.Println(err)
-  } else {
-    fmt.Println(response.StatusCode)
-    fmt.Println(response.Body)
-    fmt.Println(response.Headers)
-  }
+// RetrieveGlobalEmailStatistics demonstrates how to retrieve e-mail statistics.
+func RetrieveGlobalEmailStatistics() {
```

Comment:
So that this is not a breaking change, could you please create an alias from `Retrieveglobalemailstatistics` to `RetrieveGlobalEmailStatistics`?

Reply:
@thinkingserious I reverted the name back to the original and put godoc on the exported function which will still make golint happy and achieve the goal of 100% on goreportcard. If you prefer an alias, please re-open the conversation.